index.html

```diff
@@ -405,9 +405,9 @@ <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>If the event is <code>pointerdown</code>, the associated device is a direct manipulation device, and the target is an {{Element}},
                    then <a href="#setting-pointer-capture">set pointer capture</a> for this <code>pointerId</code> to the target element as described in <a>implicit pointer capture</a>.
 
-                <p>Fire the event to the determined target.
+                <p>Fire the event to the determined target. The user agent SHOULD treat the target as if the pointing device has moved over it for the purpose of <a href="uievents/#events-mouseevent-event-order">ensuring event ordering</a>.
 
-                <div class="note">Using the <a>pointer capture target override</a> as the target instead of the normal hit-test result may fire some boundary events. This is the same as the pointer leaving its previous target and entering this new capturing target — and if they are different targets, boundary events should be dispatched first. When the capture is released, the same scenario may happen, as the pointer is leaving the capturing target and entering the hit-test target.</div>
+                <div class="note">Using the <a>pointer capture target override</a> as the target instead of the normal hit-test result may fire some boundary events. This is the same as the pointer leaving its previous target and entering this new capturing target. When the capture is released, the same scenario may happen, as the pointer is leaving the capturing target and entering the hit-test target.</div>
```

Comment:
The requirement to dispatch boundary events is up to the ui-events spec, and the note was meant to be informative that as a result of the capture being equivalent to moving the mouse over the capture element it may (as defined by ui-events) generate boundary events. Would it be better if I changed it to read "may fire some boundary events as defined by <a href="uievents/#events-mouseevent-event-order">ui-events</a>"?

Reply:
yeah i think adding in a reference to [UI-EVENTS] would just make it clear that this *is* actually defined (somewhere else) and that we're not just handwaving it

Reply:
(which then also excuses the use of "may" in that note, as we're clearly not making any kind of normative statement since we're just pointing somewhere else)

Reply:
added the reference to `as defined by [[UI-EVENTS]]` to this PR